magma/magma
# [orc8r] Adding DNS configs to gateway cellular configs level

## Summary

- Adds gateway DNS configs, as part of the lte module
- Makes the configs part of LTE gateway cellular config, with same default values as before
- Updating orc8r/mconfigs dnsd entry and moving it to lte/mconfigs
- Network DNS configs will be deprecated in future, this just adds DNS configs at gateway level that will override those

## Test Plan

- make precommit
- Tested locally with orc8r and AGW
- Validated gateway.mconfig successfully builds on local AGW with cloud configs on Orchestrator
- Remove cellular DNS config for gateway to check that default mconfig works as expected 

## Additional Information

- [] This change is backwards-breaking

- This will override network DNS configs at LTE gateway level, I manually checked prod networks and they all seem to use default values anyway so we should be good with the change.
